doc/cygnus-ngsi/installation_and_administration_guide/deprecated_and_removed.md

```diff
@@ -30,7 +30,12 @@ Deprecated features are removed not before 3 developement sprints (usually, a de
 [Top](#top)
 
 ##<a name="section2"></a>Deprecated functionalities
-Currently, there is no deprecated functionality.
+###<a nanem="section2.1"></a>Grouping Rules
+Added after releasing version [0.5](https://github.com/telefonicaid/fiware-cygnus/releases/tag/release-0.5) (issue [107](https://github.com/telefonicaid/fiware-cygnus/issues/107)).
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1312/commits/1012fdb5693a7ed0cbd3164d46d5f10c53de5baf